src/structures/GuildChannel.js

```diff
@@ -95,13 +95,27 @@ class GuildChannel extends Channel {
    */
   get permissionsLocked() {
     if (!this.parent) return null;
-    if (this.permissionOverwrites.size !== this.parent.permissionOverwrites.size) return false;
-    return this.permissionOverwrites.every((value, key) => {
-      const testVal = this.parent.permissionOverwrites.get(key);
+
+    // Get all overwrites
+    const overwriteIds = new Set([...this.permissionOverwrites.keys(), ...this.parent.permissionOverwrites.keys()]);
+
+    // Compare all overwrites
+    return [...overwriteIds].every(key => {
+      const channelVal = this.permissionOverwrites.get(key);
+      const parentVal = this.parent.permissionOverwrites.get(key);
+
+      // Handle empty overwrite
+      // eslint-disable-next-line eqeqeq
+      if (channelVal === undefined && parentVal.deny.bitfield == 0 && parentVal.allow.bitfield == 0) return true;
```

Comment:
Is there a reason you are not using `=== 0n`?

Reply:
Hadn't initially worked when I wrote this fix, but will take another look 👍

Reply:
No idea what I did before, looks to be consistently `0n` now -- preference on `0n` vs `BitField.defaultBit`?

Reply:
We should use constants (`BitField.defaultBit`) over "magic numbers", so my preference is the latter.

Plus, in the case we somehow change the type, this would be one refactor less to do.

Reply:
Unresolving this because I noticed something:

https://github.com/discordjs/discord.js/blob/e3e466d3e56145110ba0cfcba7b3c594267d36fb/src/util/BitField.js#L164

The constant we want to compare with is `Permissions.defaultBit`, as `allow` and `deny` from `PermissionsOverwrites` are Permissions instances:

https://github.com/discordjs/discord.js/blob/e3e466d3e56145110ba0cfcba7b3c594267d36fb/src/util/Permissions.js#L141